.github/workflows/test.yml

```diff
@@ -53,10 +53,31 @@ jobs:
         run: yarn test:ci
   cypress-run:
     runs-on: ubuntu-16.04
+    strategy:
+      matrix:
+        node: [10, 12, 14]
```

Comment:
I'm not sure what benefit we get out of running the cypress tests on multiple versions of node as the tests execute in a browser environment?

Reply:
This was a request of @JacobMGEvans [see](https://discord.com/channels/808364903822917662/808364903822917665/815360342124265522)

Reply:
it doesn't hurt so happy to take as-is.

Perhaps there is more benefit to the martrix for the `jest` tests as they execute in the the node environment? (Can be seperate PR)